build/sortxtb.xslt

```diff
@@ -1,10 +1,9 @@
 <xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
-  <xsl:output method="xml" indent="yes" omit-xml-declaration="no" encoding="UTF-8" />
+  <xsl:output method="xml" indent="yes" omit-xml-declaration="no" encoding="UTF-8"/>
   <xsl:strip-space elements="*"/>
   <xsl:template match="translationbundle">
-      <!-- XSLT doesn't support adding doctype without PUBLIC or SYSTEM annotation, so add it to the doc raw -->
-      <xsl:text disable-output-escaping="yes">&lt;!DOCTYPE translationbundle&gt;
-</xsl:text>
+    <!-- XSLT doesn't support adding doctype without PUBLIC or SYSTEM annotation, so add it to the doc raw -->
+    <xsl:text disable-output-escaping="yes">&lt;!DOCTYPE translationbundle&gt;</xsl:text>
```

Comment:
I added this newline to keep the formatting the same as in the old case, but am fine with removing it